README.md

```diff
@@ -23,6 +23,12 @@ Run the following command to generate the test runner:
 ember generate ember-cli-blueprint-test-helpers
 ```
 
+If you are using `ember-cli-blueprint-test-helpers` on Node v4.x or lower, you'll want to use the `-babel` option, to add ES6 support.
```

Comment:
nopt generates some shorthands by default, so `-babel` is equal to `--babel=true` and `--babel`

I'm fine saying `--babel`, but both will work.